README.md

```diff
@@ -33,7 +33,7 @@ There is an explanation on what fields to use with what kind of contents, as wel
  phone_work       | Work phone of the user                               | Phone number in international format                           | +447474747474      | False                                | False                                | 
  birthday         | User birth date                                      | Date in ISO 8601 format                                        | 2018-10-23         | False                                | False                                | 
  work_start       | User employment start date                           | Date in ISO 8601 format                                        | 2018-10-23         | False                                | False                                | 
-
```

Comment:
What should I put instead of XXX in this table: columns `Originator import required` and `User only import required`?

Reply:
`False` in both cases, this field will be optional for all uploads.